### DISCUSSION TITLE
Suggestion to improve Quality of Life

### DISCUSSION THREAD

**Discussion by mustafatoktas** [emoacht/Monitorian#602] [Ideas]
Can you add the ability to adjust the brightness with the mouse wheel when hovering over the monitorian icon in the taskbar?

This feature is available in the Twinkle Tray application, but Twinkle Tray is quite heavy compared to Monitorian. It uses too much RAM.

I would be very happy if you add the feature I mentioned to Monitorian.

I am using monitorian, which I downloaded from Windows Store, on Windows 11 Home 23H2 22631.3672 system.
Monitorian version: 4.6.14.0

**Comment by emoacht** [comment]
Have you read https://github.com/emoacht/Monitorian/issues/174#issuecomment-1021812898 ?
